corehq/apps/ota/views.py

```diff
@@ -353,7 +353,12 @@ def _safe_int(val):
             num_unsent_forms=num_unsent_forms, num_quarantined_forms=num_quarantined_forms,
             commcare_version=commcare_version, build_profile_id=build_profile_id,
             last_heartbeat=datetime.utcnow(), modified_on=datetime.utcnow())
-        record.process_record(couch_user)
+        try:
+            record.process_record(couch_user)
+        except ResourceConflict:
+            # https://sentry.io/dimagi/commcarehq/issues/521967014/
+            couch_user = CouchUser.get(couch_user.user_id)
+            record.process_record(couch_user)
```

Comment:
Are you sure this won't cause an infinite loop? Do you know for sure that the conflict is coming from another process and not from this process itself?

Reply:
Pretty sure this is the right fix, since this was [how it was before](https://github.com/dimagi/commcare-hq/pull/26305/files#diff-291dfd3802bf0daa51e598788d5ca13bL319) I implemented this async feature.